src/garage/envs/rl2_env.py

```diff
@@ -0,0 +1,98 @@
+"""Wrapper class that is specifc for RL2."""
+import akro
+import gym
+import numpy as np
+
+from garage.envs.env_spec import EnvSpec
+
+
+class RL2Env(gym.Wrapper):
+    """Environment wrapper for RL2.
+
+    In RL2, observation is concatenated with previous action,
+    reward and terminal signal to form new observation.
+
+    Also, different tasks could have different observation dimension.
+    An example is in ML45 from MetaWorld (reference:
+    https://arxiv.org/pdf/1910.10897.pdf). This wrapper pads the
+    observation to the maximum observation dimension with zeros.
+
+    Args:
+        env (gym.Env): An env that will be wrapped.
+        max_obs_dim (int): Maximum observation dimension in the environments
```

Comment:
This is a needed feature to actually use ML45, but I think it probably belongs in a separate wrapper. We can address that in a later PR after submitting this.